commons/src/test/java/org/openmrs/eip/camel/OauthProcessorTest.java

```diff
@@ -137,15 +145,22 @@ public void process_shouldCallGetNewTokenAndSetTheHeaderIfEnabledAndTheCachedTok
 	@Test
 	public void process_shouldFailWhenTheReturnedTokenHasAnUnSupportedType() throws Exception {
 		setInternalState(processor, "isOauthEnabled", true);
-		Map<String, Object> testResponse = new HashMap();
+		Map<String, Object> testResponse = new HashMap<>();
 		testResponse.put(OauthProcessor.FIELD_TOKEN, "some-token");
 		final String type = "MAC";
 		testResponse.put(OauthProcessor.FIELD_TYPE, type);
 		when(mockProducerTemplate.requestBody(OauthProcessor.OAUTH_URI, null, Map.class)).thenReturn(testResponse);
 		assertNull(getInternalState(processor, "oauthToken"));
 		Exchange exchange = new DefaultExchange(mockCamelContext);
 		
-		assertThrows("Unsupported oauth token type: " + type, EIPException.class, () -> processor.process(exchange));
+		assertThrows(EIPException.class, () -> {
+			try {
+				processor.process(exchange);
+			}
+			catch (Exception e) {
+				throw new EIPException("Unsupported oauth token type: " + type, e);
+			}
+		}, "Unsupported oauth token type: " + type);
```

Comment:
This looks like a hacky way of testing the actual exception thrown. Any reason for doing this?

Reply:
None, Probably auto-completion/suggestion.

It should be tho;
```java
try {
  processor.process(exchange);
}
catch (Exception e) {
  assertEquals("Unsupported oauth token type: " + type, e.getMessage());
}
```